src/Simulation/Simulators.Tests/TestProjects/MicrosoftSimulatorExe/MicrosoftSimulatorExe.csproj

```diff
@@ -1,9 +1,18 @@
-<Project Sdk="Microsoft.Quantum.Sdk">
+﻿<Project Sdk="Microsoft.Quantum.Sdk">
 
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
-    <ExecutionTarget>microsoft.simulator</ExecutionTarget>
+	<OutputType>Exe</OutputType>
+	<TargetFramework>net6.0</TargetFramework>
+	<!-- we will provide our own -->
+	<IncludeQSharpCorePackages>false</IncludeQSharpCorePackages>
+	<IncludeCSharpRuntime>false</IncludeCSharpRuntime>
+	<IncludeProviderPackages>false</IncludeProviderPackages>
+	<ExecutionTarget>microsoft.simulator</ExecutionTarget>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\EntryPointDriver\Microsoft.Quantum.EntryPointDriver.csproj" />
+	<ProjectReference Include="..\..\..\QSharpCore\Microsoft.Quantum.QSharp.Core.csproj" />
+  </ItemGroup>
```

Comment:
FYI: Indentation is done with spaces in L14 and tabs in L15 (and green L4 - L10).

Reply:
updated.